src/util/mobile.hpp

```diff
@@ -190,5 +190,28 @@ namespace big::mobile
 		{
 			*scr_globals::freemode_global.at(853).as<int*>() = 1;
 		}
+
+		inline void request_gun_van()
+		{
+			auto local_pos      = self::pos;
+			auto forward_vector = ENTITY::GET_ENTITY_FORWARD_VECTOR(self::ped);
+			Vector3 spawn_point;
+
+			if (MISC::FIND_SPAWN_POINT_IN_DIRECTION(local_pos.x,
+			        local_pos.y,
+			        local_pos.z,
+			        forward_vector.x,
+			        forward_vector.y,
+			        forward_vector.z,
+			        25.f,
+			        &spawn_point))
+			{
+				*scr_globals::gun_van.as<Vector3*>() = spawn_point;
+
+				return g_notification_service->push_success("Request Service", "The Gun Van is now spawned near your location, keep in mind that this is local and cannot be seen by other players.");
+			}
+
+			g_notification_service->push_warning("Request Service", "Couldn't find a suitable spawn point for the Gun Van, try moving to a different location.");
```

Comment:
Missing from translations.

Reply:
I approved by accident, anyways still need them added to the translations.

Reply:
These need to be translated? I can't seem to find any other notifications that are translated. For example the notification: "Mechanic is not ready to deliver a vehicle right now." is not in "en_US".

Reply:
Just search for g_notification_service->push in the project, you'll find plenty of results making use of translations.

https://github.com/search?q=repo%3AYimMenu%2FYimMenu+g_notification_service-%3Epush&type=code

Reply:
Fixed now.